project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_quickpost.scss

```diff
@@ -31,6 +31,10 @@
       right: 7px;
 
       label {
+        color: transparent;
```

Comment:
I tried this, but this way, when clicking on `<i>` icon, `<a>` link above the label is not activated:

   `<a href="javascript://" ng-click="browseFiles('c4m-related-document')"></a>
    <span class="body-attachment-link">
        <label for="c4m-related-document">
         <i class="fa fa-paperclip"></i>
       </label>`
    </span>
  </div>